app/assets/javascripts/editor.js.erb

```diff
@@ -574,14 +575,10 @@ $(function() {
     //    output_mode_is_streaming = false;
     //}
     if (!colorize) {
-      if(output.stdout != ''){
-        element.append(output.stdout)
-      }
-
-      if(output.stderr != ''){
-        element.append('There was an error: StdErr: ' + output.stderr);
-      }
-
+      var stream = _.sortBy([output.stderr || '', output.stdout || ''], function(stream) {
+        return stream.length;
+      })[1];
```

Comment:
This code was fixed in the meantime, please integrate the pushed changes in the meantime.